Added the always_run option in excepthook document page (#978)
getsentry/sentry-docs: src/collections/_documentation/platforms/python/default-integrations.md

any exception that is unhandled will be reported to Sentry automatically.  Exceptions
raised interactive interpreter sessions will not be reported.

### Options

You can pass the following keyword arguments to `ExcepthookIntegration()`:

* `always_run`:
    
    ```python

>     #repl

Reviewer: How about this?

```sh
$ python
>>> import sentry_sdk
>>> from sentry_sdk.integrations.excepthook import ExcepthookIntegration
>>> sentry_sdk.init(..., integrations=[ExcepthookIntegration(always_run=True)])
>>> raise Exception("I will become an error")
```

Author: I'm not good at the document. This is much better. :) Thank you for both correction. I'm updating now.

    raise Exception('I can catch an error through in Repl.')
    ```

>     The Sentry does not catch error in repl when it is always_run's default value. If the default value is set to `True`, errors in repl can be captured. (`sentry_sdk.init(integrations=[ExcepthookIntegration(always_run=True)])`)

Reviewer: ```suggestion
    By default, the SDK does not capture errors occuring in the REPL (`always_run=False`).
```

  >>> raise Exception("I will become an error")
  ```

>   By default, the SDK does not capture errors occuring in the REPL (`always_run=False`).

Reviewer 2: ```suggestion
  By default, the SDK does not capture errors occurring in the REPL (`always_run=False`).
```

Author: ops, now fixing .. :) Bad typo

Reviewer 2: No worries! This looks great. :) Thanks for all your work!

any exception that is unhandled will be reported to Sentry automatically.  Exceptions
raised interactive interpreter sessions will not be reported.

### Options

You can pass the following keyword arguments to `ExcepthookIntegration()`:

* `always_run`:
    
    ```bash
    $ python
    >>> import sentry_sdk
    >>> from sentry_sdk.integrations.excepthook import ExcepthookIntegration
    >>> sentry_sdk.init(..., integrations=[ExcepthookIntegration(always_run=True)])
    >>> raise Exception("I will become an error")
    ```
    By default, the SDK does not capture errors occuring in the REPL (`always_run=False`).

>   The default is `False`.

Reviewer: This line duplicates information. Also please check the indentation level, I think the code block is indented with three spaces
